main.go

```diff
@@ -124,16 +142,124 @@ func NewExporter(server string, timeout time.Duration) *Exporter {
 		),
 		evictions: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "items_evicted_total"),
-			"Number of valid items removed from cache to free memory for new items.",
+			"Total number of valid items removed from cache to free memory for new items.",
 			nil,
 			nil,
 		),
 		reclaimed: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "items_reclaimed_total"),
-			"Number of times an entry was stored using memory from an expired entry.",
+			"Total number of times an entry was stored using memory from an expired entry.",
 			nil,
 			nil,
 		),
+		malloced: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "malloced_bytes"),
+			"Number of bytes of memory allocated to slab pages.",
+			nil,
+			nil,
+		),
+		itemsNumber: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "slab", "items_number"),
+			"Number of items currently stored in this slab class.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsAge: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "slab", "items_age_seconds"),
+			"Number of seconds the oldest item has been in the slab class.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsEvicted: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "slab", "items_evicted"),
```

Comment:
I'm very sure this and all following `memcached_slab_items_*` metrics are counters and should also have the `_total` suffix. I'll double check our current statistics again, but the wording in the protocol documentation https://github.com/memcached/memcached/blob/master/doc/protocol.txt#L791 "Number of times ..." indicates that as well.